### PR TITLE
[IOTDB-5326] Add cluster_name parameter

### PR DESCRIPTION
**Pull request by CRZbulabula**
Add cluster_name parameter for which the user can distinguish different clusters. The cluster_name is configurable as shown below:

image

I also enhance ClusterIT to test when the cluster_name are inconsist. And the user docs are updated by the way.

### CODE REVIEW

File: confignode/src/assembly/resources/conf/iotdb-confignode.properties [hunk 1]
```diff
@@ -17,6 +17,14 @@
 # under the License.
 #
 
+####################
+### Cluster Configuration
+####################
+
+# Used for indicate cluster name and distinguish different cluster.
+# Datatype: string
+cluster_name=defaultCluster
```
Comment (ericpai) [inline confignode/src/assembly/resources/conf/iotdb-confignode.properties:26]: All the keys in iotdb-confignode.properties have `cn_` prefix. It's better to change `cluster_name` to `cn_cluster_name`, or just move it to iotdb-commom.properties ?

File: server/src/assembly/resources/conf/iotdb-datanode.properties [hunk 2]
```diff
@@ -17,6 +17,14 @@
 # under the License.
 #
 
+####################
+### Cluster Configuration
+####################
+
+# Used for indicate cluster name and distinguish different cluster.
+# Datatype: string
+cluster_name=defaultCluster
```
Comment (ericpai) [inline server/src/assembly/resources/conf/iotdb-datanode.properties:26]: Both iotdb-datanode.properties and iotdb-confignode.properties have this parameter, which violates our [design](https://apache-iotdb.feishu.cn/docx/doxcnjzB2xd9sScIlkNYJUJA1zb) about the configuration files. Move it to iotdb-common.properties instead.

**Review by ericpai — CHANGES_REQUESTED** [review]

File: server/src/assembly/resources/conf/iotdb-datanode.properties (hunk 2 above)
Comment (CRZbulabula) [reply server/src/assembly/resources/conf/iotdb-datanode.properties]: Fixed~

File: confignode/src/assembly/resources/conf/iotdb-confignode.properties (hunk 1 above)
Comment (CRZbulabula) [reply confignode/src/assembly/resources/conf/iotdb-confignode.properties]: I've moved it into the iotdb-common.properties~

**Review by ericpai — APPROVED** [review]

**Review by Beyyes — APPROVED** [review]
LGTM

**Pull request merged**